### ISSUE TITLE
Failure to install using Go modules

### ISSUE THREAD

**Issue by horgh** [magnusbaeck/logstash-filter-verifier#66]
I'm trying to install this via go install when I have Go modules enabled, and the install fails like so:

```
$ go install github.com/magnusbaeck/logstash-filter-verifier
go: finding github.com/magnusbaeck/logstash-filter-verifier latest
go: finding github.com/breml/logstash-config/ast latest
go: finding github.com/op/go-logging latest
go: finding github.com/mitchellh/packer/common/json latest
go: finding github.com/mitchellh/packer/common latest
go: github.com/mitchellh/packer@v1.3.4: parsing go.mod: unexpected module path "github.com/hashicorp/packer"
go: error loading module requirements
$ go version
go version go1.11.5 linux/amd64
```

I realise you provide releases and other ways to build/install the program, but it'd be nice to be able to install this way as well! I was hoping to vendor the program in a repo using Go modules.

I suspect it may have something to do with github.com/mitchellh/packer being renamed, but I'm not sure.

Thank you for creating this tool by the way!

**Comment by magnusbaeck** [comment]
As of commit d4664fd89e7e23cde870fddf04bdd745824c9517 LFV has been converted to use modules so hopefully this has been resolved. Could you test again?

**Comment by horgh** [comment]
Using `make` works with modules! `go install github.com/magnusbaeck/logstash-filter-verifier` fails with `version` not being defined, but that's probably expected?

**Comment by magnusbaeck** [comment]
Ah, of course. I never thought about that. I've addressed that in commit 5d46a16. After pushing that I realized I broke `make release-tarballs` so it'll take some extra fixes before I can release v1.5.2 but I consider this issue fixed (finally).
